README.md

```diff
@@ -11,58 +11,27 @@ User application for Agoric Inter Protocol--Vaults, BLD Boost, Liquidations, etc
 
 ### Setup
 
-1. Download and build the latest copy of `agoric-sdk`, including Agoric's Cosmic SwingSet.
+1. Run the agoric local chain
 
    ```sh
-   cd agoric-sdk
-   yarn && yarn build
-   yarn link-cli ~/bin/agoric # or use any local dir in $PATH (e.g. ~/.local/bin/agoric)
+   docker run -d -p 26657:26657 -p 1317:1317 -p 9090:9090 ghcr.io/agoric/agoric-3-proposals:main
    ```
 
-   Test that `agoric` works with:
-
-   `agoric --version`
-
-   For Cosmic SwingSet (in `agoric-sdk`):
-
-   ```sh
-   cd packages/cosmic-swingset
-   make
-   ```
-
-   Test that Cosmic SwingSet tools work with:
-
-   `agd --help`
-
-2. (One-time) Define keys expected by the startup scripts\*
-
-   ```sh
-   agd keys add gov1 --keyring-backend=test
-   agd keys add gov2 --keyring-backend=test
-   ```
-
-   \*If you have access to the mnemonics, use `--interactive` to get a prompt to import them. Without that it will create new keys. You can save the seed phrase somewhere, or even create a keplr account with it for testing. You can use `agd keys list --keyring-backend=test` to check which keys you've created.
-
-3. Start a local chain with psm, vaults, etc.
-
-   ```
-   cd packages/inter-protocol && scripts/start-local-chain.sh
-   ```
-
-4. Go the the [wallet](https://github.com/Agoric/wallet-app) repository and start a local client server:
+2. Clone and run wallet app
```

Comment:
Wallet app isn't necessary anymore, we can remove this step

Reply:
@samsiegart Regarding the process of setting up the web wallet, I have a question. Currently, we set up the web wallet and proceed with our inter dApp test cases for updating/creating/deleting a vault. We're using the web wallet because it has enough ATOMs to serve as collateral. However, I'm wondering if this step might be outdated or if there's another way for provisioning the wallet that we should follow.

Reply:
The web wallet interfaces with the same account/smart wallet on-chain as the dapp-inter and Keplr, so the ATOM balance isn't a factor. I assume you're using it to provision the smart wallet by paying 10 BLD (and also suggest the local testnet to Keplr). The provisioning step can be done inside dapp-inter via this feature: https://github.com/Agoric/dapp-inter/pull/219

Adding the local testnet to Keplr I don't think is possible yet with dapp-inter. It should probably be added as a feature if not, but not a blocker for testing if you can do it with the web wallet already.